mysqld-mixin/README.md

```diff
@@ -0,0 +1,26 @@
+# MySQLd Mixin
+
+_This is a work in progress. We aim for it to become a good role model for alerts
+and dashboards eventually, but it is not quite there yet._
```

Comment:
Let's drop this, it smells of the :construction: banner. :smile_cat:

Reply:
:+1: